### PR TITLE
Map some blocks, fluids and block entities

### PR DESCRIPTION
**Pull request by i509VCB**

### CODE REVIEW

File: mappings/net/minecraft/fluid/FlowingFluid.mapping [hunk 1]
```diff
@@ -1,4 +1,5 @@
-CLASS net/minecraft/class_3609 net/minecraft/fluid/BaseFluid
+CLASS net/minecraft/class_3609 net/minecraft/fluid/FlowingFluid
```
Comment (liach) [inline mappings/net/minecraft/fluid/FlowingFluid.mapping]: flowing or flowable? so is this like flowing water/lava excluding water/lava source blocks?

Comment (i509VCB) [reply mappings/net/minecraft/fluid/FlowingFluid.mapping]: Well rather it's a fluid that __*could*__ flow, this includes the source blocks also

Comment (liach) [reply mappings/net/minecraft/fluid/FlowingFluid.mapping]: Then probably `FlowableFluid`, flowing sounds like it only includes flowing water/lava

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/block/InfestedBlock.mapping [hunk 2]
```diff
@@ -1,11 +1,17 @@
 CLASS net/minecraft/class_2384 net/minecraft/block/InfestedBlock
 	FIELD field_11178 regularBlock Lnet/minecraft/class_2248;
 	FIELD field_11179 REGULAR_TO_INFESTED Ljava/util/Map;
-	METHOD <init> (Lnet/minecraft/class_2248;Lnet/minecraft/class_4970$class_2251;)V
+	METHOD <init> <init> (Lnet/minecraft/class_2248;Lnet/minecraft/class_4970$class_2251;)V
+		COMMENT Creates an infested block
```
Comment (modmuss50) [inline mappings/net/minecraft/block/InfestedBlock.mapping:5]: We really need to slove this engima issue

**Review by modmuss50 — APPROVED** [review]

File: mappings/net/minecraft/block/entity/CommandBlockBlockEntity.mapping [hunk 3]
```diff
@@ -8,9 +8,11 @@ CLASS net/minecraft/class_2593 net/minecraft/block/entity/CommandBlockBlockEntit
 	METHOD method_11037 setNeedsUpdatePacket (Z)V
 		ARG 1 needsUpdatePacket
 	METHOD method_11038 setPowered (Z)V
+		ARG 1 powered
 	METHOD method_11039 getCommandBlockType ()Lnet/minecraft/class_2593$class_2594;
 	METHOD method_11040 getCommandExecutor ()Lnet/minecraft/class_1918;
 	METHOD method_11041 setAuto (Z)V
+		ARG 1 automatic
```
Comment (Juuxel) [inline mappings/net/minecraft/block/entity/CommandBlockBlockEntity.mapping]:
```suggestion
		ARG 1 auto
```

**Review by Juuxel — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/block/InfestedBlock.mapping (hunk 2 above)
Comment (i509VCB) [reply mappings/net/minecraft/block/InfestedBlock.mapping:5]: I assume its the `<init> <init>` thing?

**Review by Juuxel — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Review by Runemoro — APPROVED** [review]

**Pull request merged**